PureBasic.sublime-syntax

```diff
@@ -415,3 +394,35 @@ contexts:
       captures:
         0: variable.other.constant.purebasic
         1: punctuation.definition.variable.purebasic
+
+  operators_all:
+    - include: operators_arithmetic
+    - include: operators_assignment
+    - include: operators_comparison
+    - include: operators_logical
+    - include: operators_bitwise
```

Comment:
I used to insert a blank line between top level contexts for improved readability. If you are amending your branch because of the broken syntax definition reference, can you add this, too?

Reply:
This comment applies to the other operator contexts as well.

Reply:
> I used to insert a blank line between top level contexts for improved readability. If you are amending your branch because of the broken syntax definition reference, can you add this, too?

I removed them because I though they might be the cause why the **Context Backtrace** is not working with the syntax — but now I think that backtrace functionality simply stopped working a few dev-updates ago, i.e. that it's an experimental features that is now broken, since I noticed that it stopped working with the native syntaxes too.

I'll try to restore the deleted empty lines, if I can't recover them for some reason I'll close the PR though.

Reply:
Original empty lines restored. But again, after force pushing the workflow build is not triggered again, so you'll have to trigger it manually.